api/ServiceWorkerGlobalScope.json

```diff
@@ -1422,6 +1422,55 @@
           }
         }
       },
+      "periodicsync_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerGlobalScope/periodicsync_event",
```

Comment:
@ddbeck Note that this is the URL where the event SHOULD be. I am moving it in https://github.com/mdn/content/pull/4057